ui/src/main/webapp/tsconfig.json

```diff
@@ -1,18 +1,24 @@
 {
   "compilerOptions": {
     "target": "es5",
-    "module": "commonjs",
+    "module": "es2015",
+    "moduleResolution": "node",
     "sourceMap": true,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "removeComments": false,
     "noImplicitAny": false,
+    "inlineSources": true,
+    "rootDir": "./",
     "outDir": "../../../target/tscoutput"
   },
   "compileOnSave": false,
   "buildOnSave": false,
   "awesomeTypescriptLoaderOptions": {
     "forkChecker": true,
     "useWebpackText": true
+  },
+  "angularCompilerOptions": {
+    "debug": true
   }
 }
```

Comment:
Do we need those changes in tsconfig?
If they are there only for AOT, then IMHO we don't need it there since it is already in tsconfig-production.